CogniteSdk/test/fsharp/Assets.fs

```diff
@@ -556,7 +595,7 @@ let ``Count assets matching multi Label ORfilter is Ok`` () = task {
 
 
     // Assert
-    test <@ count > 0 @>
+    test <@ count >= 6 @>
```

Comment:
Can we instead test that the result count is greater than 0 and that all result items either have label1 or label2. That would be more robust than just checking if the result count is greater than 5 (we don't know what labels the result items then have).

Reply:
For the count request test I believe we're better off setting a minimum result count (i.e. 2), as any result less than that indicates something's up with our test data. We also can't validate the item count without performing an additional filter request as well.